docs/additionalpackages.md

```diff
@@ -49,7 +49,30 @@ The controller domain is not an all-purpose Linux system. It must remain minimal
 * Avoid any software that may interfere with the existing
 * Avoid software that widens the attack surface on your hosts
 
-### 5. Ask before
+### 5. Known Security Risks
+
+#### Libreswan
+
+If you are using encrypted tunnels using `openvswitch-ipsec` and `libreswan`, for example through [Xen Orchestra's SDN Controller](https://xen-orchestra.com/docs/sdn_controller.html) there are security advisories you need to know about, there are 2 CVEs that are affecting our current libreswan version:
+- [CVE-2023-38712](https://libreswan.org/security/CVE-2023-38712/CVE-2023-38712.txt): Invalid IKEv1 repeat IKE SA delete causes crash and libreswan to restart
+- [CVE-2023-38710](https://libreswan.org/security/CVE-2023-38710/CVE-2023-38710.txt): Invalid IKEv2 REKEY proposal causes libreswan to restart
+
+Patches for these are not really backportable, we therefore kept it as is and target updating packages in the next major release.
+
+You can find information about all Libreswan CVEs on their [security page](https://libreswan.org/security/).
+
+##### Resulting security issue
+
+The `pluto` daemon may be crashed by malformed IKE (both v1 and v2) delete/notify requests, resulting in a DDoS on the keying service. If the daemon is restarted in loop quickly enough, this could as well lead to a DDoS of the whole host.
```

Comment:
Is it DDoS (Distributed Denial of Service), or just DoS?

Reply:
I think it is indeed DoS.

Reply:
I changed it.